content/guides/http/go/net_http/client.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "Client"
+date: 2018-11-26T22:22:31-07:00
+draft: false
+aliases: [/integrations/http/go/net_http/client]
+weight: 1
+logo: /images/gopher.png
+---
+
+- [Introduction](#introduction)
+- [Traces](#traces)
+- [Metrics](#metrics)
+- [End-to-end example](#end-to-end-example)
+- [Viewing traces](#view-traces)
```

Comment:
I think link should be ```(#viewing-traces)```, same for below line ```(#viewing-metrics)```.

Reply:
Oh, good catch, thanks @mayurkale22!